docs/concepts/fields.md

```diff
@@ -801,11 +801,14 @@ Read more about JSON schema customization / modification with fields in the [Cus
 ## The `computed_field` decorator
 
 ??? api "API Documentation"
-    [`pydantic.fields.computed_field`][pydantic.fields.computed_field]<br>
+    [`computed_field`][pydantic.fields.computed_field]<br>
 
-The `computed_field` decorator can be used to include `property` or `cached_property` attributes when serializing a
-model or dataclass. This can be useful for fields that are computed from other fields, or for fields that
-are expensive to computed (and thus, are cached).
```

Comment:
I decided to remove the part about:

> This can be useful for fields that are computed from other fields, or for fields that
are expensive to computed (and thus, are cached).

As this isn't really specific to `computed_field` but rather `property`/`cached_property`. Let me know if that suits you.

Reply:
Thank you, this alteration makes very clear that `computed_field` performs no validation. Thanks so much!

Reply:
I'd rather keep this information bc it helps to contextualize usage patterns for folks. Could we instead add a note specifying that no validation is involved in order to clarify this murky point?